### ISSUE TITLE
User agent string causes certain web servers to fail request

### ISSUE THREAD

**Issue by sureshven** [Alamofire/Alamofire#1452]
The 3.4.2 update of Alamofire creates a user agent string with an extra close paren in Manager.swift line 95. This causes certain web servers to fail when validating the user agent string.

`return "\(executable)/\(bundle) (\(appVersion)/\(appBuild)); \(osNameVersion))"`

should be:

`return "\(executable)/\(bundle) (\(appVersion)/\(appBuild); \(osNameVersion))"`

**Comment by tobiasoleary** [comment]
Are you saying it should be this?
```
return "\(executable)/\(bundle) (\(appVersion)/\(appBuild); \(osNameVersion)"
```
Both return statements looked the same too me.

**Comment by tobiasoleary** [comment]
Nevermind.

**Comment by cnoon** [comment]
Thank you for reporting @sureshven. This issue has been resolved in PR #1456 and commit 68ddac4d. It's also been cherry picked into the `swift2.3` and `swift3` branches and will be released very soon.

Cheers. 🍻
